djstripe/management/commands/djstripe_process_specific_events.py

```diff
@@ -0,0 +1,72 @@
+from django.core.management.base import BaseCommand
+
+from ... import models
+from ... import settings as djstripe_settings
+from ._djstripe_command_utils import VerbosityAwareOutputUtils, parse_string_to_bool
+
+
+class Command(BaseCommand, VerbosityAwareOutputUtils):
+    """Sync and process specific events specified by ID.
+
+    Note: this is only guaranteed to retrieve events that are at most 30 days old based
+    on the current limitation of stripe's events API.
+    See: https://stripe.com/docs/api/events
+    """
+
+    help = (
+        "Sync and process specific events specified by ID."
+        " Note: this is only guaranteed retrieve events that are at most"
+        " 30 days old based on the current limitation of stripe's events API."
+        " See:  https://stripe.com/docs/api/events"
+    )
+
+    def add_arguments(self, parser):
+        """Add the argument to get the IDs."""
+        parser.add_argument(
+            "ids", nargs="*", help="A space separated list of Event IDs to sync."
+        )
+        parser.add_argument(
+            "--livemode",
+            type=parse_string_to_bool,
+            help=(
+                "Overrides the STRIPE_LIVE_MODE setting for"
+                " determining which API key to use."
+            ),
+        )
+
+    def handle(self, *args, **options):
+        """Try to process the specific Events that were requested."""
+        self.set_verbosity(options)
+
+        self.output("Syncing events..")
+
+        live_mode = options["livemode"]
+        if live_mode is None:
+            live_mode = djstripe_settings.STRIPE_LIVE_MODE
+
+        api_key = djstripe_settings.get_default_api_key(live_mode)
```

Comment:
What's the reasoning behind having livemode for this command but not the other?

Reply:
I missed that the API key was a parameter that could be provided to `api_list()`. I have it here because there's no "fetch_single" function built in that doesn't also require a Event instance (not just class).

I'll make them consistent at least, but is there a preferred method? The `api_list` function defaults it to `settings.STRIPE_SECRET_KEY` but some places use `settings.get_default_api_key()`. Is there a preferred method?

Reply:
Fair enough. 

Yeah, I don't think we need to be able to choose livemode (though I'm open to argument).  So we can just use STRIPE_SECRET_KEY, which will be correct for our current mode.

Reply:
OK, I will drop the parameter and change over to using `STRIPE_SECRET_KEY`.